packages/jsii-diff/lib/classes-ifaces.ts

```diff
@@ -296,3 +314,49 @@ function isCompatibleArgumentType(original: reflect.TypeReference, updated: refl
   // Input can never be void, so no need to check
   return isSuperType(updated, original, updated.system);
 }
+
+/**
+ * Verify assignability to supertypes
+ *
+ * For every base type B of type T, someone could have written:
+ *
+ * ```
+ * const variable: B = new T();
+ * ```
+ *
+ * This code needs to be valid in the updated assembly, so for each
+ * B an updated type B' needs to exist in the new assembly which is
+ * still a supertype of T'.
+ */
+function assignableToAllBaseTypes(original: reflect.ReferenceType, updated: reflect.ReferenceType): Analysis {
+  for (const B of baseTypes(original)) {
```

Comment:
```suggestion
  for (const base of baseTypes(original)) {
```

Reply:
Yeah. Well it was `B` on purpose to match the docstring just above it.

 But sure.